include/powsybl/iidm/Connectable.hpp

```diff
@@ -24,8 +24,16 @@ class Network;
 
 class Connectable : public Identifiable, public MultiVariantObject {
 public:
+    Connectable(const Connectable& connectable) = delete;
+
+    Connectable(Connectable&& connectable) noexcept;
```

Comment:
I'm not sure we have to provide this body and let the compiler generate it

Reply:
If I remember well, you cannot use a default implementation because you inherit from virtual base classes.